OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -137,6 +137,107 @@ - (void) _setupEventAlarmFromProperties: (NSDictionary *) properties
     }
 }
 
+- (int) _updateFromAttendeeMAPIProperties: (NSArray *) recipients
+                                 withRole: (NSString *) role
+                                 outParam: (BOOL *) organizerIsSet
+{
+  NSDictionary *dict;
+  NSString *attEmail;
+  iCalPerson *person;
+  iCalPersonPartStat newPartStat;
+  NSNumber *flags, *trackStatus;
+  int i, effective;
+
+  effective = 0;
+  for (i = 0; i < [recipients count]; i++)
```

Comment:
I agree that this is bad code, but out of scope. I'd change `i` to `size_t`, create a variable to avoid calling method on each iteration and I don't want to read more lines because ignorance is happiness